.circleci/config.yml

```diff
@@ -218,7 +218,7 @@ jobs:
                   keys:
                       - v2-golden-images-<< pipeline.parameters.current_golden_images_hash >>-hcm-
                       - v2-golden-images-main-hcm-
-                      - v2-golden-images-main-classic-lightest-medium-ltr-
+                      - v2-golden-images-main-classic-light-medium-ltr-
```

Comment:
I think we can remove this all together. It was to give a backstop when we first added HCM support.